pkg/gitreceive/k8s_util.go

```diff
@@ -108,9 +108,12 @@ func slugbuilderPod(
 	pod.Spec.Containers[0].Name = slugBuilderName
 	pod.Spec.Containers[0].Image = image
 
+	if _, ok := env["DEIS_DISABLE_CACHE"]; !ok {
```

Comment:
do we want the cache to be enabled by default? also it would be nice to see it documented on how to use it.